<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="MainWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>831</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ControlWidget">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>串口</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>串口号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="com_list"/>
          </item>
          <item>
           <widget class="QPushButton" name="refresh_com">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="open_com">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>打开</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>191</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>OSK</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>176</width>
         <height>83</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="osk_enable">
          <property name="text">
           <string>OSK enable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="external_osk_enable">
          <property name="text">
           <string>External OSK enable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="external_osk_pin_en">
          <property name="text">
           <string>External OSK pin activate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>IO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QCheckBox" name="auto_io_update">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto IO Update</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="SweepWidget">
     <attribute name="title">
      <string>Sweep</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>321</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="TOP">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="H1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Sweep Frequency 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Sweep Frequency 1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="change_value_0"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>MHz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="change_value_1"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>MHz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="H1_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Rising Step Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Falling Step Size</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="change_value_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>MHz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="change_value_3"/>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>MHz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="H1_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Rising Sweep Ramp Rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Falling Sweep Ramp Rate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="change_value_4"/>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>uS </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="change_value_5"/>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>uS </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="H1_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Frequency Jump Point 0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Frequency Jump Point 1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="change_value_6"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>MHz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="change_value_7"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>MHz</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>180</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>DRCTL</string>
      </property>
      <widget class="QCheckBox" name="set_auto_drctl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>set_auto_drctl</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="set_drctl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>72</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>set_drctl</string>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="set_enable_sweep">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable Sweep</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="set_enable_frequency_jump">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable Frequency Jump</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>260</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>DWELL</string>
      </property>
      <widget class="QCheckBox" name="set_dewll_high">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>set_Dwell_High</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="set_dwell_low">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>set_Dwell_Low</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="set_io_update">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>更新寄存器</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ModulesWidget">
     <attribute name="title">
      <string>Modules</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
